packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -34,9 +34,14 @@ const TradingAppCard = ({
                 <TradigPlatformIconProps icon={icon} size={48} />
             </div>
             <div className={classNames('trading-app-card__details', { 'trading-app-card--divider': has_divider })}>
-                <Text className='title' size='xs' line_height='s'>
-                    {sub_title}
-                </Text>
+                <div>
+                    <Text className='title' size='xs' line_height='s'>
+                        {sub_title}
+                    </Text>
+                    {short_code_and_region && (
+                        <span className='trading-app-card__details__short-code'>{short_code_and_region}</span>
```

Comment:
Change this to `<Text>` component and use the props for that component such as size and weight and ...